src/scenekit.cs

```diff
@@ -1038,6 +1185,9 @@ interface SCNSceneLoadingOptions {
 	[Mac (10,8), iOS (8,0)]
 	delegate bool SCNNodePredicate (SCNNode node, out bool stop);
```

Comment:
is it used elsewhere ? or does `SCNNodeHandler` replace it everywhere ?

if the later case let's put it under `!XAMCORE_4_0`